charts/k8ssandra/templates/cassandra/cassdc.yaml

```diff
@@ -207,7 +185,7 @@ spec:
         volumeMounts:
           - name: {{ include "medusa.configMapName" . }}
             mountPath: /etc/medusa
-          - name: server-config
+          - name: cassandra-config
```

Comment:
I think you are right. The `cassandra-config` volume mount will eventually have all the bits we need, but it's not until the `cassandra` container kicks in that it copies the updated configs from `/config` over to `/etc/cassandra`.

I need to test this some more to make sure I get this bit right. The idea here is to have all of the config dumped into the `cassandra-config` volume mount, as my intention is to only ever use that volume mount for `/etc/cassandra`, with the exception of the first init container, since I don't want to clobber what's in the management api image until after I've copied them out.